_events/2017-04-09-this-bridge-called-my-back.md

```diff
@@ -0,0 +1,9 @@
+---
+identifier: this-bridge-called-my-back
```

Comment:
`identifier` and `reading_identifier` seem redundant to me. Should these values also be wrapped in quotes?